apps/src/dropletUtils.js

```diff
@@ -276,6 +287,18 @@ exports.generateDropletModeOptions = function (dropletConfig) {
   populateModeOptionsFromConfigBlocks(modeOptions, { blocks: exports.dropletBuiltinConfigBlocks });
   populateModeOptionsFromConfigBlocks(modeOptions, dropletConfig);
 
+  setTitlesToFuncNamesForDocumentedBlocks(modeOptions);
+
   return modeOptions;
 };
 
+/**
+ * Returns a set of all blocks
+ * @param dropletConfig
+ * @returns {Object.<String, Object>}
+ */
+exports.getAllDropletBlocks = function (dropletConfig) {
+  return $.extend(exports.dropletGlobalConfigBlocks,
+    exports.dropletBuiltinConfigBlocks,
+    dropletConfig.blocks);
+};
```

Comment:
I wonder if this would be better written as 
```
$.extend(exports.dropletGlobalConfigBlocks, exports.dropletBuiltinConfigBlocks, dropletConfig.blocks)
return exports.dropletGlobalConfigBlocks
```
I think this makes it a little more clear what's happening. Also, do you know the difference between globalconfigblocks and builtingconfigblocks? Seems like those could use comments.